README.md

```diff
@@ -336,6 +336,7 @@ designed to support multiple output formats.
 Currently, the following report formats are supported (reporter names are case-insensitive):
 
 * [Amazon OSS Attribution Builder](https://github.com/amzn/oss-attribution-builder) document (*experimental*, `-f AmazonOssAttributionBuilder`)
+* [Attribution Document (PDF)](https://github.com/eclipse/antenna) (`-f AttributionDocumentPDF`)
```

Comment:
This should be marked as "experimental" seeing and missing features.

Where is the documentation, how can we test this feature?

Reply:
How exactly the reporter works in detail will be described in the upcoming documentation (see #2674). 

But for now, you can just run the reporter with the `-f AttributionDocumentPDF` option.

Reply:
> This should be marked as "experimental" seeing and missing features.

IMO "experimental" sounds too much as if the existing features are uncertain to work, which is not the case. We rather have some smaller missing feature, like the support for multiple projects, but that is an inherent limitation in Antenna. However, this could probably be easily worked around by writing out one document per project.